ghost/portal/src/components/Frame.styles.js

```diff
@@ -399,7 +399,7 @@ const FrameStyles = `
 }
 
 .gh-portal-powered a:hover {
-    color: #15171A;
+    color: var(--grey0);
```

Comment:
This logo is marketing material which should be the same everywhere. These values should remain hardcoded.